lib/screens/mobile/mobile_home.dart

```diff
@@ -399,7 +524,45 @@ class _MobileHomeScaffoldState extends State<MobileHomeScaffold> with TickerProv
         return Padding(
           padding: const EdgeInsets.symmetric(horizontal: 110),
           child: MaterialButton(
-            onPressed: () {},
+            onPressed: () async {
+              try {
+                List<String> argsData = _argsProviders[index].controllers.map((controller) => controller.text).toList();
+                Map<String, dynamic> kWarValues = {};
+                for (final map in _kwargsProviders[index].tableData) {
+                  String key = map["key"];
+                  dynamic value = map["value"];
+                  kWarValues[key] = value;
+                }
+                Map<String, dynamic> formattedResult = kWarValues;
+                var session = await connect(
+                  _tabData[index].linkController.text,
+                  _tabData[index].realmController.text,
+                  _tabData[index].selectedSerializer,
+                );
+
+                Future.delayed(const Duration(seconds: 1), () {
+                  session.call(_tabData[index].topicProcedureController.text, args: argsData, kwargs: formattedResult);
+                  _tabData[index].linkController.clear();
+                  _tabData[index].realmController.clear();
+                  _tabData[index].selectedSerializer = "";
+                  _argsProviders[index].controllers.clear();
+                  _kwargsProviders[index].tableData.clear();
+                });
```

Comment:
And display the args and kwargs of result on screen

```suggestion
                 var result = await session.call(_tabData[index].topicProcedureController.text,
                    args: argsData, kwargs: formattedResult);
                _tabData[index].linkController.clear();
                _tabData[index].realmController.clear();
                _tabData[index].selectedSerializer = "";
                _argsProviders[index].controllers.clear();
                _kwargsProviders[index].tableData.clear();
```